idd/Energy+.idd.in

```diff
@@ -35833,11 +35833,13 @@ ZoneHVAC:PackagedTerminalAirConditioner,
    A11, \field Cooling Coil Object Type
         \required-field
         \type choice
+        \key Coil:Cooling:DX
         \key Coil:Cooling:DX:SingleSpeed
         \key Coil:Cooling:DX:VariableSpeed
         \key CoilSystem:Cooling:DX:HeatExchangerAssisted
         \note Select the type of Cooling Coil.
-        \note Only works with Coil:Cooling:DX:SingleSpeed or
+        \note Only works with Coil:Cooling:DX or
+        \note Coil:Cooling:DX:SingleSpeed or
```

Comment:
Disclaimer: I know none of this is really enforced on E+ side, and is merely used for IDF Editor, and that we care about it a lot more on the OpenStudio's side.

The 'Cooling Coil Name' field is missing `\object-list CoilCoolingDX` so that it also matches the `Coil:Cooling:DX` 

```diff
   A11, \field Cooling Coil Object Type
        \required-field
        \type choice
        \key Coil:Cooling:DX
        \key Coil:Cooling:DX:SingleSpeed
        \key Coil:Cooling:DX:VariableSpeed
        \key CoilSystem:Cooling:DX:HeatExchangerAssisted
        \note Select the type of Cooling Coil.
        \note Only works with Coil:Cooling:DX or
        \note Coil:Cooling:DX:SingleSpeed or
        \note CoilSystem:Cooling:DX:HeatExchangerAssisted or
        \note Coil:Cooling:DX:VariableSpeed.
   A12, \field Cooling Coil Name
        \required-field
        \type object-list
        \object-list CoolingCoilsDXSingleSpeed
        \object-list CoolingCoilsDXVariableSpeed
+       \object-list CoilCoolingDX
        \note Needs to match a DX cooling coil object.
```

---

(and technically `CoolingCoilsDXSingleSpeed` matches the `Coil:Cooling:DX:SingleSpeed` and the `CoilSystem:Cooling:DX:HeatExchangerAssisted` but also the `Coil:Cooling:DX:SingleSpeed:ThermalStorage`)